.travis.yml

```diff
@@ -1,28 +1,27 @@
 language: go
 
 go:
-  - 1.6
-  - 1.7
+  - 1.9
```

Comment:
Is go1.9+ mandatory or recommended?
i.e. Ensure no build failure in go 1.7 enviornment

Reply:
go1.9+ is recommended and build is not failing with go 1.7.